brianlong/www.validators.app
# 182695701-fix-charts-when-duplicated-id

#### What's this PR do?
- on validators/show charts should not get data from other network when account is duplicated

#### How should this be manually tested?
- run tests
- try changing network of some validator history displayed on the charts (validators/show)
- it should disappear from the chart

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182695701)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
